src/main/java/com/salesforce/dataloader/controller/Controller.java

```diff
@@ -458,23 +467,35 @@ private static String getProductName() {
         return APP_NAME + " " + APP_VERSION;
     }
 
-    public static synchronized void initLog() throws FactoryConfigurationError {
-
+    public static synchronized void initLog() throws FactoryConfigurationError, ControllerInitializationException {
         // init the log if not initialized already
         if (Controller.isLogInitialized) {
             return;
         }
 
-        File logConfXml = Paths.get(System.getProperty("user.dir"), "configs", LOG_CONF_OVERRIDE).toFile();
-        if (logConfXml.exists()) {
-        	System.setProperty("log4j.configurationFile", logConfXml.getAbsolutePath());
+    	try {
+    		initStaticVariable();
+    	} catch (ControllerInitializationException ex) {
+    		System.out.println("Controller.initLog(): Unable to initialize Controller static vars: " + ex.getMessage());
```

Comment:
We probably want to pipe this into log4j

Reply:
We can't pipe it because log4j is not initialized yet. That's why I am going for System.out.println().